src/DotLiquid.Tests/Util/ObjectExtensionMethodsTests.cs

```diff
@@ -0,0 +1,31 @@
+using System.Collections.Generic;
```

Comment:
Nit: best to keep all the using statements in a single block (no empty line) and organize them with System* first, then everything else, each sub-group alphabetically ordered.

Reply:
Done